### PR TITLE
Steer Linux users towards the supported service manager

### PR DESCRIPTION
**Pull request by zeha**
### Short description
Describe the usually expected configuration (systemd as pid1) first, then refer to init scripts.

### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: docs/running.rst [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 =====================
 
-PowerDNS is normally controlled via a SysV-style init.d script, often
-located in ``/etc/init.d`` or ``/etc/rc.d/init.d``. For Linux
-distributions with systemd, a service file is provided (either in the
-package or in the contrib directory of the tarball).
+On Linux, PowerDNS is controlled by a systemd service called
+``pdns.service``. The service definition file should be installed by
+the binary package, and can also be found in the tarball
+(``pdns.service.in`` template file).
```
Comment (Habbie) [inline docs/running.rst:7]: can you put new sentences on single lines? Over time we hope to get rid of all line breaks in the middle of sentences.

**Review by Habbie — CHANGES_REQUESTED** [review]
wonderful! one formatting nit (for multiple places)

File: docs/running.rst (hunk 1 above)
Comment (zeha) [reply docs/running.rst]: Changed. I hope the new format is what you described?

Comment (Habbie) [reply docs/running.rst]: yes, thanks!

**Pull request merged**